estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,40 @@
+from odoo import api, fields, models, exceptions
+
+
+class PropertyOffer(models.Model):
+    _name = 'estate.property.offer'
+    _description = 'Property Offer'
+
+    price = fields.Float()
+    status = fields.Selection([("accepted", "Accepted"), ("refused", "Refused")], copy=False)
+    partner_id = fields.Many2one("res.partner", string="Partner", required=True)
+    property_id = fields.Many2one("estate.property", string="Property", required=True)
+    validity = fields.Integer(string="Validity (days)", default=7,)
+    date_deadline = fields.Date("Deadline", compute="_compute_deadline", inverse="_inverse_deadline")
+
+    def _inverse_deadline(self):
+        for record in self:
+            record.validity = (record.date_deadline - fields.Date.today()).days
+
+    @api.depends("validity")
+    def _compute_deadline(self):
+        for record in self:
+           if record.create_date:
+               record.date_deadline = fields.Date.add(record.create_date, days=record.validity)
+           else:
+               record.date_deadline = fields.Date.add(fields.Date.today(), days=record.validity)
```

Comment:
```suggestion
        for offer in self:
           create_date = offer.create_date or fields.Date.today()
           record.date_deadline = fields.Date.add(create_date, days=record.validity)
```
a bit more concise :smile: